src/Components/animation/DisplayedElement.js

```diff
@@ -99,217 +99,119 @@ class DisplayedElement extends React.Component {
 
     switch (this.state.stateFromProps.id) {
       case 'defense':
-        if (this.state.isMoving)
-          return (
-            <Animated.View
-              {...this.panResponder.panHandlers}
-              style={[
-                panStyle,
-                styles.defense,
-                { height: this.state.stateFromProps.playerRadius },
-                { width: this.state.stateFromProps.playerRadius },
-                { borderRadius: this.state.stateFromProps.playerRadius },
-                { left: 0 },
-                { top: 0 },
-              ]}
-              key={this.state.stateFromProps.id}
-            >
-              <Text style={styles.defenseText}>{this.state.stateFromProps.number}</Text>
-            </Animated.View>
-          );
-        else
-          return (
-            <Animated.View
-              {...this.panResponder.panHandlers}
-              style={[
-                {
-                  transform: [
-                    {
-                      translateX: this.state.stateFromProps.interpolateX,
-                    },
-                    {
-                      translateY: this.state.stateFromProps.interpolateY,
-                    },
-                  ],
-                },
-                styles.defense,
-                { height: this.state.stateFromProps.playerRadius },
-                { width: this.state.stateFromProps.playerRadius },
-                { borderRadius: this.state.stateFromProps.playerRadius },
-                { left: 0 },
-                { top: 0 },
-              ]}
-              key={this.state.stateFromProps.id}
-            >
-              <Text style={styles.defenseText}>{this.state.stateFromProps.number}</Text>
-            </Animated.View>
-          );
-
-      case 'offense':
-        if (this.state.isMoving)
-          return (
-            <Animated.View
-              {...this.panResponder.panHandlers}
-              style={[
-                panStyle,
-                styles.offense,
-                { height: this.state.stateFromProps.playerRadius },
-                { width: this.state.stateFromProps.playerRadius },
-                { borderRadius: this.state.stateFromProps.playerRadius },
-                { left: 0 },
-                { top: 0 },
-              ]}
-              key={this.state.stateFromProps.id}
-            >
-              <LinearGradient
-                colors={[theme.GRADIENT_FIRST_COLOR, theme.GRADIENT_SECOND_COLOR]}
-                style={[
-                  styles.gradient,
-                  {
-                    height: this.state.stateFromProps.playerRadius,
-                    width: this.state.stateFromProps.playerRadius,
-                    borderRadius: this.state.stateFromProps.playerRadius,
+        return (
+          <Animated.View
+            {...this.panResponder.panHandlers}
+            style={[
+              this.state.moving
```

Comment:
@ZacharieALES instead of duplicating the whole rendered component I just applied the condition to the changing style